src/AspNetCoreModuleV2/RequestHandlerLib/FileOutputManager.cpp

```diff
@@ -7,7 +7,10 @@
 #include "exceptions.h"
 #include "debugutil.h"
 
-FileOutputManager::FileOutputManager()
+FileOutputManager::FileOutputManager() :
+    m_hLogFileHandle(INVALID_HANDLE_VALUE),
```

Comment:
This is unrelated to this change. Will make it a separate PR.

Reply:
Handle wrapper? :)

Reply:
Sure.